src/main/java/com/dev/cinema/controllers/mappers/ShoppingCartMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.controllers.mappers;
+
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.dto.ShoppingCartResponseDto;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartMapper {
+    private final TicketMapper mapper;
+
+    public ShoppingCartMapper(TicketMapper mapper) {
+        this.mapper = mapper;
+    }
+
+    public ShoppingCartResponseDto fromEntityToDto(ShoppingCart cart) {
```

Comment:
```suggestion
    public ShoppingCartResponseDto mapToDto(ShoppingCart cart) {
```
use uniform method names